CMake preset configuration step fails - can't find benchmark pkg

**Describe the bug**
CMake configuration failing on develop branch due to unfound benchmark package. Occurs on various presets (GUI-nix, GUI-Debug-nix, CLI-nix etc).

```CMake Error at CMakeLists.txt:327 (find_package):
  By not providing "Findbenchmark.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "benchmark", but CMake did not find one.

  Could not find a package configuration file provided by "benchmark" with
  any of the following names:

    benchmarkConfig.cmake
    benchmark-config.cmake

  Add the installation prefix of "benchmark" to CMAKE_PREFIX_PATH or set
  "benchmark_DIR" to a directory containing one of the above files.  If
  "benchmark" provides a separate development package or SDK, be sure it has
  been installed.
```

**To Reproduce**
1. Checkout `develop`
2. from top level directory run `cmake --preset <PRESET>; cmake --build build --target <TARGET>...` 

**Expected behaviour**
CMake preset configuration should succeed, results in building Dissolve

**Operating System:**
Linux, nixOS

--- a/.github/workflows/benchmark.yml
+++ b/.github/workflows/benchmark.yml
@@ -59,7 +59,7 @@ jobs:
         if: ${{ github.ref_name == 'develop' }}
         with:
             tool: 'googlecpp'
-            output-file-path: all_benchmark_results.json
+            output-file-path: build/all_benchmark_results.json
             github-token: ${{ secrets.GITHUB_TOKEN }}
             auto-push: true
             comment-on-alert: true

--- a/flake.nix
+++ b/flake.nix
@@ -25,6 +25,7 @@
           antlr4
           antlr4.runtime.cpp
           antlr4.runtime.cpp.dev
+          gbenchmark
           cmake
           cli11
           fmt_8

